domain/model/service/doc.go

```diff
@@ -0,0 +1,22 @@
+// Copyright 2024 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+// Package service contains the services required for interacting wit the
+// underlying models within a Juju controller. This includes the management of
+// model's and the respective information for each model.
+//
+// # Model Status
+// A model maintains and can be in exactly one status state at any given time.
+// We consider a model's status as purely a user interface value and not a
+// programmatic one that should be used for gating decisions on with the
+// business logic of the controller.
```

Comment:
Suggest slight reword
```
// We consider a model's status as purely a user interface value and not a
// programmatic one that should be used for informing the operation of
// the business logic of the controller.
```

Plus, no need for first line - isn't it self evident that a model (or unit or app or machine) can be expected to only have one status at a time?

Reply:
Yeah probably. I just had the old implementation in my head which technically could result in multiple states at once.